babel.config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["@babel/preset-env"]
+}
```

Comment:
Please, don't ignore new lines at the end of file. Once I faced some issue on production with a configuration file that didn't have proper ending.